test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -1212,6 +1212,13 @@ def _validate_argument_spec(self, docs, spec, kwargs, context=None):
                     code='parameter-alias-repeated',
                     msg=msg
                 )
+            if not context and arg == 'state':
+                bad_states = {'list', 'info'} & set(data.get('choices', set()))
```

Comment:
Looks like the CI failure is caused by the set literal under Python 2.6:
```suggestion
                bad_states = set('list', 'info') & set(data.get('choices', set()))
```

Reply:
(if that doesn't work, change `&` with calling `set.intersection()`)